src/features/projects/components/PopupProject.tsx

```diff
@@ -37,6 +39,11 @@ export default function PopupProject({
     '%';
 
   const projectDetails = project.properties;
+
+  const handleRedirect = () => {
+    router.push(`https://donate.plant-for-the-planet.org/?to=${projectDetails.id}`);
```

Comment:
Same as https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1318/files#r773669034